src/open_inwoner/templates/pages/profile/me.html

```diff
@@ -106,6 +106,33 @@ <h2 class="h2 title" id="notifications">{% trans "Voorkeuren voor meldingen" %}
     <section class="profile-section profile-section__overview">
         <h2 class="h2 title" id="profile-overview">{% trans "Overzicht" %}</h2>
         <div class="profile-cards card-container card-container--columns-3">
+            {# Selected categories #}
+            {% if view.config.selected_categories %}
+                <a href="{% url 'profile:categories' %}" class="card card--compact card--stretch" id="profile-section-contacts">
+                    <div class="card__body">
+                        <div class="profile__link">
+                            <p class="h4"><span class="link link__text">{% trans "Mijn Interessegebieden" %}</span></p>
+
+                            {% render_list %}
+                                <span class="profile-list ellipsis">
+                                    {% for name in selected_categories %}
+                                        {% list_item text=name compact=True strong=False %}
+                                    {% empty %}
+                                        <p class="p list_paragraph">{% trans "U heeft geen interesses gekozen." %}</p>
```

Comment:
Not sure if this works the way it should - when selecting Onderwerpen, it does save, but: after going back and deselecting everything + submit, it won't let me completely empty the options since they are required:

<img width="1051" alt="Screenshot 2024-02-12 at 11 24 26" src="https://github.com/maykinmedia/open-inwoner/assets/118177951/d0637780-9c1d-47ed-b0b5-a3601a3eed09">

Reply:
I changed this to allow emptying the selected categories